link_linux.go

```diff
@@ -2440,6 +2464,9 @@ func linkSubscribeAt(newNs, curNs netns.NsHandle, ch chan<- LinkUpdate, done <-c
 				continue
 			}
 			for _, m := range msgs {
+				if m.Header.Flags&unix.NLM_F_DUMP_INTR != 0 && cberr != nil {
```

Comment:
even if for initial list would return dump interrupted there would be additional notifications from kernel (i think).
not sure we want this here.

Reply:
Possibly ... what additional notifications are you thinking of, and where would they go?